src/Projekt/Projekt.fsproj

```diff
@@ -81,16 +57,54 @@
   <Target Name="BeforeBuild">
   </Target>
   -->
-  <ItemGroup>
-    <Templates Include="$(SolutionDir)\templates\**\*.*"/>
-  </ItemGroup>
   <Target Name="AfterBuild">
-    <Copy
-        SourceFiles="@(Templates)"
-        DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)"
-        />
+    <Copy SourceFiles="@(Templates)" DestinationFolder="$(OutputPath)\templates\%(Templates.RecursiveDir)" />
```

Comment:
No it absolutely does, but msbuild understands the wildcard perfectly well, I was hoping that the over-eager VS check was implemented separately.